data.sql

```diff
@@ -31,6 +31,40 @@ INSERT INTO species (name) VALUES
 ('Digimon');
 
 
+INSERT INTO vets (name,age, date_of_graduation) VALUES 
+('William Tatcher',45,'2000-04-23'),
+('Maisy Smith',26,'2019-01-17'),
+('Stephanie Mendez',64,'1981-05-4'),
+('Jack Harkness',45,'2008-06-08');
+
+INSERT INTO specializations (vid,sid) VALUES 
+(1,1),
```

Comment:
[OPTIONAL ] 
- You could also use colunm names to insert the data, rather than hardcoded data.
Example for line 41 below:
```
INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'William Tatcher'), (SELECT id FROM species WHERE name = 'Pokemon'));
```